### DISCUSSION TITLE
Stacked and grouped bar chart

### DISCUSSION THREAD

**Discussion by AnteVukusic** [ant-design/ant-design-charts#2754] [General]
Hello, I was wondering is there a way to build this kind of graph using antd design charts.

![image](https://github.com/user-attachments/assets/3f2f226d-60c5-4004-b257-17aa4f094043)

This is my data set:

savings_account_name - bar item
date - x axis
amount - y axis
total_amount - height of the bar
saved_amount - height of the bar

It should be grouped by date, and on each date there should be a bar for each savings_account_name in which there is stacked total_amount and saved_amount

**Comment by lxfu1** [comment]
https://ant-design-charts.antgroup.com/examples/statistics/column/#group-and-stack
